# Массив значений компенсаторов
uint8[8] status